ramls/request.json

```diff
@@ -123,6 +123,9 @@
           "type": "string"
         }
       },
+      "enumeration": {
```

Comment:
Properties should have a description, please add one to this property

Reply:
Hmm, I don't know what it means and there is not a description in item's schema  too 😄

Reply:
I asked Charlotte, and I suggest we use `the volume or issue number for the item`

Reply:
Done